docs/superpowers/plans/2026-04-28-pending-count-ipc.md

```
@@ -0,0 +1,930 @@
+### Status file (waybar-friendly)
+
+The daemon writes `$XDG_RUNTIME_DIR/mac-notifications-status.json` on every
+state change. The file includes a `count` field:
+
+```bash
+jq -r .count "$XDG_RUNTIME_DIR/mac-notifications-status.json"
+```
+
+Combined with `SIGRTMIN+11` waybar refresh, this is zero-cost polling for
+status-bar widgets.
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)